From 3c6c44e4c79b446491a6a21184da540a3cd7e3f7 Mon Sep 17 00:00:00 2001
From: James Slagle <jslagle@redhat.com>
Date: Wed, 19 Feb 2014 06:59:27 -0500
Subject: [PATCH 15/15] Move 99-neutronclient under neutron-source-install

This is source install specific, so it should live under
neutron-source-install.

Change-Id: I2804d05e28f0119819c2b2aea14a468f42be251e
---
 elements/neutron/install.d/99-neutronclient                        | 7 -------
 elements/neutron/install.d/neutron-source-install/99-neutronclient | 7 +++++++
 2 files changed, 7 insertions(+), 7 deletions(-)
 delete mode 100755 elements/neutron/install.d/99-neutronclient
 create mode 100755 elements/neutron/install.d/neutron-source-install/99-neutronclient

diff --git a/elements/neutron/install.d/99-neutronclient b/elements/neutron/install.d/99-neutronclient
deleted file mode 100755
index 8542412..0000000
--- a/elements/neutron/install.d/99-neutronclient
+++ /dev/null
@@ -1,7 +0,0 @@
-#!/bin/bash
-set -eux
-
-# Due to trunk neutron relying on trunk neutronclient, we need
-# trunk neutronclient - https://bugs.launchpad.net/neutron/+bug/1280941
-/opt/stack/venvs/neutron/bin/pip install -U git+https://git.openstack.org/openstack/python-neutronclient
-
diff --git a/elements/neutron/install.d/neutron-source-install/99-neutronclient b/elements/neutron/install.d/neutron-source-install/99-neutronclient
new file mode 100755
index 0000000..8542412
--- /dev/null
+++ b/elements/neutron/install.d/neutron-source-install/99-neutronclient
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -eux
+
+# Due to trunk neutron relying on trunk neutronclient, we need
+# trunk neutronclient - https://bugs.launchpad.net/neutron/+bug/1280941
+/opt/stack/venvs/neutron/bin/pip install -U git+https://git.openstack.org/openstack/python-neutronclient
+
-- 
1.8.5.3

